dbms/src/Storages/KVStore/MultiRaft/PreHandlingTrace.h

```diff
@@ -22,14 +22,35 @@
 
 namespace DB
 {
+enum class ReadFromStreamError
```

Comment:
I think `PrehandleError` or `PrehandleStatus` is a more reasonable name here

Reply:
Renamed to `PrehandleTransformStatus`, because there is a `PrehandleResult` for the whole prehandle process